### PR TITLE
chore: return 404 when projectid not found

### PR DESCRIPTION
**Pull request by daveleek**
## About the changes
Returns Not Found on create and get project api tokens when given a project id that doesn't exist

## Discussion points
- This is an extra lookup per execution of the endpoint

### CODE REVIEW

File: src/lib/routes/admin-api/project/api-token.ts [hunk 1]
```diff
@@ -143,6 +149,11 @@ export class ProjectApiTokenController extends Controller {
     ): Promise<void> {
         const { user } = req;
         const { projectId } = req.params;
+        const project = await this.projectService.getProject(projectId);
+        if (!project) {
+            res.status(404).end();
```
[the remark this answers is not in the document]
Comment (daveleek) [reply src/lib/routes/admin-api/project/api-token.ts:154]: Thank you @nunogois - that's a good point! I'll try that, might actually be what's happening and why it's working

**Review by nunogois — APPROVED** [review]
LGTM. Maybe I would rephrase the comment a bit, but that's just me.
`await this.projectService.getProject(projectId); // Validates that the project exists`
